_includes/schemas/trackers_response.json

```diff
@@ -0,0 +1,73 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "properties": {
```

Comment:
If `to` and `from` addresses are given, they will be returned as properties as well. Should we add this?